modules/unsupported/ogr/ogr-core/src/main/java/org/geotools/data/ogr/OGRDataStoreFactory.java

```diff
@@ -112,6 +112,17 @@
                     false,
                     DEFAULT_EVICTOR_TESTS_PER_RUN);
 
+    public static final boolean DEFAULT_PRIME_DATASOURCE = false;
+
+    /** Number of connections checked during a single evictor run * */
```

Comment:
does this comment comes from a copy and paste? :)

Reply:
Doh!!!